docs/content/how-to/distribute-hosted-cluster-workloads.md

```diff
@@ -28,6 +28,41 @@ In addition:
       role.kubernetes.io/infra: "" 
 ```
 
+## Scheduling Topology Options 
+
+Cluster Service Providers may choose how hosted control planes are isolated or co-located. The three different options are:
+
+ - Shared Everything
+ - Shared Nothing 
+ - Dedicated Request Serving
+
```

Comment:
I would add some wording to the effect that this is a spectrum... 
shared nothing -> shared some (non-request-serving) -> shared everything